libs/eo/metering-points/shell/src/lib/eo-metering-point-table.component.ts

```diff
@@ -35,6 +37,8 @@ import { EoMeteringPoint, EoMeteringPointsStore } from './eo-metering-points.sto
     MatSortModule,
     WattBadgeComponent,
     WattSpinnerComponent,
+    WattCardComponent,
```

Comment:
Why is WattCardComponent added here?

Reply:
Removed. check latest commit:

[6bc893d](https://github.com/Energinet-DataHub/greenforce-frontend/pull/1967/commits/6bc893d766cf2c001a8977e1f087b77e2e8e38a7)